roles/openldap/tasks/password_policy.yml

```diff
@@ -0,0 +1,65 @@
+- name: check if ppolicy is part of the schema
+  shell: ldapsearch -Y EXTERNAL -H ldapi:/// -s sub
+    -b cn=config objectclass=olcSchemaConfig | grep ppolicy
+  register: ppolicy_out
+  ignore_errors: yes
+  changed_when: False
+
+- name: add password policy schema
+  shell: ldapadd -Y EXTERNAL -H ldapi:/// -f  /etc/ldap/schema/ppolicy.ldif
+  when: ppolicy_out.stdout.count("ppolicy,") < 1
+
+- name: create policy container
+  shell: ldapadd -x -D {{openldap.root_dn }} -w {{ openldap.root_password }} 
+    -f {{ openldap.config_dir }}/ppolicy_container.ldif
```

Comment:
This looks like it needs to be whitespaced over a bit, same with other shell tasks in this pr. For multiline the second line should start at the same level as the previous line

```
  shell: ldapadd -x -D {{openldap.root_dn }} -w {{ openldap.root_password }} 
         -f {{ openldap.config_dir }}/ppolicy_container.ldif
```